object.c

```diff
@@ -1413,18 +1433,14 @@ true_and(VALUE obj, VALUE obj2)
 
 /*
  *  call-seq:
- *     true | obj   -> true
- *
- *  Or---Returns <code>true</code>. As <i>obj</i> is an argument to
- *  a method call, it is always evaluated; there is no short-circuit
- *  evaluation in this case.
+ *    true | object -> true
  *
- *     true |  puts("or")
- *     true || puts("logical or")
+ *  Returns +true+:
```

Comment:
I think it would be good to mention that this does not short circuit (meaning `object` is evaluated). For example:

```ruby
true | raise # => Raises RuntimeError.
```

Which is different from the short circuiting operator:

```ruby
true || raise # => true
```

Reply:
Added.